app/controllers/app.py

```diff
@@ -337,14 +337,18 @@ def post(self, project_id):
 
         app_data = request.get_json()
 
+        app_data['project_id'] = project_id
```

Comment:
Why do we need this?

Reply:
So that `project_id` is stored in the App model, as it is needed by the database and the schema.
So instead of the `project_id` being passed to the schema from the request JSON object form, its added to the object from the parameter `project_id`.